test/bugs/82-emojis.js

```diff
@@ -30,7 +30,7 @@ describe("Bug #82: db.query errors when parsing emojis ", function () {
     return this.db.query(query)
     .bind(this)
     .spread(function (result) {
-      result.should.eql(1);
+      result.should.eql('1');
```

Comment:
if `.eql` is now stricter, we should just use `.equal` when comparing primitives